picard/ui/logview.py

```diff
@@ -25,6 +27,8 @@
 
 class LogViewCommon(PicardDialog):
 
+    default_verbosity = {log.LOG_INFO, log.LOG_WARNING, log.LOG_ERROR, log.LOG_DEBUG}
```

Comment:
I am not sure how much sense this makes. Generally debug includes all the levels above it. So if debug is checked all errors, info and warning logs should come too.
This is default behaviour in almost all loggers.

Reply:
That's not the behaviour of the logger here, but the behaviour of the log viewer: each log message is set to a certain level (and that's the default behaviour of almost all loggers), whether you want to log or display a certain level is up to you.
For example, i may want to write only debug messages to one debug file, and the rest in a common file, or each level in a separate file, that's not defined by the logger, but by the log handler.

The idea here is to reduce the view to what is interesting, if i know the thing i'm looking for is a debug message, i want to hide the other levels. And this is possible here.